frontend/src/routes/Governance/discovered/grouping.ts

```diff
@@ -169,10 +192,26 @@ export function grouping(): {
         }
       }
 
+      let responseAction: string
+      const allResponseActionsList: string[] = Array.from(allResponseActions)
+
+      if (allResponseActions.size === 1) {
+        responseAction = allResponseActionsList[0]
+      } else if (
+        allResponseActions.size === 2 &&
+        allResponseActions.has('inform') &&
+        allResponseActions.has('enforce')
+      ) {
+        responseAction = 'inform/enforce'
+      } else {
```

Comment:
What is this case? any example?

Reply:
If there are multiple enforcement actions (e.g. `deny` and `warn` becomes `deny/warn` in the UI).

Reply:
gotcha